docs/source/common/loggers.rst

```diff
@@ -217,6 +230,31 @@ The :class:`~pytorch_lightning.loggers.WandbLogger` is available anywhere except
             # Option 2 for specifically logging images
             self.logger.log_image(key="generated_images", images=[some_img])
 
+To visualize using wandb in a jupyter notebook environment use the following magic line command:
```

Comment:
this is all nice and useful, but we need to be careful not to replicate wandb documentation into our own docs. just as a note for the future.